lib/rules/no-regex-spaces.js

```diff
@@ -23,23 +23,64 @@ module.exports = {
     create: function(context) {
         var sourceCode = context.getSourceCode();
 
-        return {
+        /**
+         * Validate regular expressions
+         * @param {ASTNode} node node to validate
+         * @param {string} value regular expression to validate
+         * @returns {void}
+         * @private
+         */
+        function regexCheck(node, value) {
```

Comment:
Can you rename each function to begin with "check"? Our conventions always have a verb as the first word in a function name.